File: OCH_Tao/ABC/A/364.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
input.shift();
let cnt = false;
let ans = false;
for (const i of input) {
  if (ans) {
    console.log("No");
    process.exit();
  }
  else if (cnt && i === "sweet") {
    ans = true;
  } else if (i === "sweet") {
    cnt = true;
  } else {
    cnt = false;
  }
}
console.log("Yes");

File: OCH_Tao/ABC/A/364.py
N = int(input())
cnt = False
X = False
S = []
for i in range(N):
  S.append(input())
for i in S:
  if X:
    print("No")
    break
  elif cnt and i=="sweet":
    X=True
  elif i=="sweet":
    cnt=True
  else:
    cnt=False
else:
  print("Yes")